### DISCUSSION TITLE
Inverse Search Emacs 28.1 & SumatraPDF v3.4.6

### DISCUSSION THREAD

**Discussion by HolgerWitte** [sumatrapdfreader/sumatrapdf#2829] [Q&A]
I use Emacs 28.1 & SumatraPDF v3.4.6 64-bit Windows 10 Pro for writing phd-thesis in latex (TeX Live 2022).

After installing the programs (see above) I try to set up a reverse search (jumping from the PDF to the editor). The search from the editor into the PDF already works. I wrote the following code in SumatraPDF-settings.txt:

`InverseSearchCmdLine = “C:\Program Files\Emacs\emacs-28.1\bin\emacsclientw.exe” -n +%l “%f”`

and in emacs

```
(setq TeX-source-correlate-mode t)
(setq TeX-source-correlate-method 'synctex)
(setq TeX-view-program-list
   '(("Sumatra PDF" ("\"C:/Program Files/SumatraPDF/SumatraPDF.exe\" -reuse-instance"
                      (mode-io-correlate " -forward-search %b %n ") " %o"))))
(eval-after-load 'tex
  '(progn
     (assq-delete-all 'output-pdf TeX-view-program-selection)
     (add-to-list 'TeX-view-program-selection '(output-pdf "Sumatra PDF"))))
(server-start)
;;(add-hook 'LaTeX-mode-hook (lambda () (add-to-list 'TeX-command-list `("Latexmk" ,(concat "latexmk -pdf -bibtex -synctex=1 %s") TeX-run-TeX nil t :help "Run Latexmk"))))
```

Unfortunately, the InverseSearch does not work. I get the error message: "Error while executing the reverse search command. Please check the setting."

The forward search works.

What am I doing wrong?

Thanks and greetings
Holger

**Comment by GitHubRulesOK** [comment]
hmm
your command `\bin\emacsclientw.exe” -n +%l “%f”` looks ok
 some users of 3.4.6 ***installed and a compressed synctex.gz***  are experiencing problems that do not happen with uncompressed .synctex OR the portable version.

see https://github.com/sumatrapdfreader/sumatrapdf/discussions/2640 and https://github.com/sumatrapdfreader/sumatrapdf/issues/2642
